sql/database.sql

```diff
@@ -5129,6 +5129,7 @@ CREATE TABLE `openemr_postcalendar_categories` (
   `pc_cattype` INT( 11 ) NOT NULL COMMENT 'Used in grouping categories',
   `pc_active` tinyint(1) NOT NULL default 1,
   `pc_seq` int(11) NOT NULL default '0',
+  `aco_spec` VARCHAR(63) NOT NULL default 'encounters|notes',
```

Comment:
I have not tested but otherwise it looks good to me. Perhaps @bradymiller can help with the PSR2 conversion. :)